geode-core/src/main/java/org/apache/geode/internal/cache/tier/sockets/AcceptorImpl.java

```diff
@@ -458,7 +458,8 @@ public AcceptorImpl(int port, String bindHostName, boolean notifyBySubscription,
             this.serverSock.bind(new InetSocketAddress(getBindAddress(), port), backLog);
             break;
           } catch (SocketException b) {
-            if (!treatAsBindException(b) || System.currentTimeMillis() > tilt) {
+            if (isGatewayReceiver || !treatAsBindException(b)
```

Comment:
See my other comment - I think that this retry logic is protecting us against a specific case when the server restarts and encounters a bind exception, so it is probably wise to leave this file how it was.

Reply:
OK. It's fair. I think it will not encounter that easily. I will remove the fix for this, including my new test.

Reply:
Thanks Gester, sorry for leading you astray ;).